package.json

```diff
@@ -86,6 +86,7 @@
     "@typechain/truffle-v5": "^2.0.2",
     "@typechain/web3-v1": "^1.0.0",
     "@types/chai": "^4.2.11",
+    "@types/ethereumjs-abi": "^0.6.3",
```

Comment:
nit: Is there a reason why we need a new dependency for this? From the docs, `web3` already has utility methods for this: https://web3js.readthedocs.io/en/v1.2.0/web3-eth-abi.html#encodeparameter

Reply:
Great suggestion, I didn't know about this method.